examples/frustum_pointnet/dataset_utils.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+NUM_HEADING_BIN = 12
+NUM_SIZE_CLUSTER = 8
+NUM_OBJECT_POINT = 512
```

Comment:
strange variable names. Maybe something like: NUM_HEAD_BINS, NUM_CLUSTERS, NUM_OBJECTS? Please change adequately. Or maybe I am not understanding what you this variables describe.